py/util.py

```diff
@@ -42,9 +42,18 @@ def run(command, cwd=None, env=None, dryrun=False):
   """
   logging.info("Running: %s \ncwd=%s", " ".join(command), cwd)
 
+  # In case the release is done from a non-linux machine
+  # we enforce correct GOOS and GOARCH
+  extra_envs = {
+    "GOOS": "linux",
+    "GOARCH": "amd64"
+    }
+
```

Comment:
We could delegate the responsability of setting these to the user, but it's a little bit cumbersome. For example if you are making modification to the dashboard's backend, you will first test the changes directly using `go run` , so using the machine native architecture, and then want to try this in cluster. Which means you have to change `GOOS` and `GOARCH` all the time. These makes things more convenient.